### ISSUE TITLE
Displaying table with one custom row

### ISSUE THREAD

**Issue by hdarby** [evrencoskun/TableView#289]
My company has been using TableView for a long time and over the course of my application's lifetime we have had to customize the look of rows/columns/cells for special circumstances. In general, we have had success doing this. In this case, however, I am completely stuck as to how to proceed.

The circumstance in question:

We display a table full of data that is laid out in a homogenous way 95% of the time. In a few cases, we have to make once row in the table appear dramatically different than the other rows. In this instance, we need to display a line of text spanning the entire width of the table, instead of displaying the regular 3 - 12 columns worth of data. There are typically 5 - 16 rows in these tables.

I am able to correctly identify the row and the cell for which the data needs to be displayed. I can't figure out how to set the cell width correctly so that that one cell knows what width it should have. I will mention that we have another customization for this table where certain columns expand and contract to show the contained data in the cells in different representations, so there is no fixed width for the entire table (if the data in the table extends beyond the view boundary, the table scrolls horizontally).

So, in short, how can I customize the tableview so that one row contains a single cell whereas the other rows contain several columns worth of data? Our current implementation ends up displaying the cells with their alignments out of whack. I think if I can set the cell width for the one row to match that of the others this will be solved.

**Comment by evrencoskun** [comment]
Hi @hdarby 

Unfortunately, Merging Cells is not supported by TableView. :/

**Comment by MGaetan89** [comment]
Could this be a duplicate of #96?
